Simplified/NYPLAccountSignInViewController.m

```diff
@@ -590,6 +591,17 @@ + (void)authorizeUsingExistingBarcodeAndPinWithCompletionHandler:(void (^)())han
 
 #pragma mark -
 
+- (void)scanLibraryCard
+{
+  [NYPLBarcode presentScannerWithCompletion:^(NSString * _Nullable resultString) {
+    if (resultString) {
+      self.usernameTextField.text = resultString;
+      [self.PINTextField becomeFirstResponder];
+      self.loggingInAfterBarcodeScan = YES;  //Prevent text from clearing
```

Comment:
from clearing under what conditions?

Reply:
I actually went ahead and removed this comment. I feel that it's common to do a file-wide search for a variable when trying to understand its purpose, and this variable is only used twice. There is a nice description in the spot where it's used as a conditional.